src/fmu/sumo/sim2sumo/tables.py

```diff
@@ -156,7 +157,10 @@ def get_table(
         print("------------------")
         print(SUBMOD_DICT[submod]["doc"])
         print("------------------")
```

Comment:
Is this `print_help` thing used?

Reply:
Well, if it is used I don't know, but it is intended to be there. It is for people to get help on setting up res2df, but I see that the rationale for having it there get's thinner and thinner. And it is also cluncy and not very elegant. So we should probably get rid of it. But not down I believe. Should have a deprectation warning for a while in case anybody has ever used it..